src/Middleware/OutputCaching/src/IOutputCacheStore.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.OutputCaching;
+
+/// <summary>
+/// Represents a store for cached responses.
+/// </summary>
+public interface IOutputCacheStore
+{
+    /// <summary>
+    /// Evicts cached responses by tag.
+    /// </summary>
+    /// <param name="tag">The tag to evict.</param>
+    /// <param name="token">Indicates that the operation should be cancelled.</param>
+    ValueTask EvictByTagAsync(string tag, CancellationToken token);
```

Comment:
Should all the `CancellationToken` parameters be renamed from `token` to `cancellationToken`?
I know the some of the `Caching` APIs used `token` but the majority uses `cancellationToken`.

<img width="766" alt="image" src="https://user-images.githubusercontent.com/4009570/174887952-d1e42d69-8f11-4eb2-9ce2-25c259894565.png">


```suggestion
    ValueTask EvictByTagAsync(string tag, CancellationToken cancellationToken);
```